bears/configfiles/NginxBear.py

```diff
@@ -0,0 +1,40 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement \
+    import DistributionRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable='nginx',
+        output_format='regex',
+        output_regex=r'\[(?P<severity>.+)\](?P<message>.+)'
+                     r':(?P<line>\d+)',
+        severity_map={'emerg': RESULT_SEVERITY.MAJOR,
+                      'alert': RESULT_SEVERITY.MAJOR,
+                      'warn': RESULT_SEVERITY.NORMAL},
+        use_stderr=True)
+class NginxBear:
+    """
+    Checks syntax of nginx configuration files using `nginx -tc` command.
+    """
+
+    LANGUAGES = {'nginx'}
+    REQUIREMENTS = {
+        DistributionRequirement(
+            apt_get='nginx',
```

Comment:
nice.  The new build system has landed, and you will need to re-run `.ci/generate_bears_metadata.py`, and then re-run `moban` , and it should update `.travis.yml` for you.